staging/src/k8s.io/controller-manager/pkg/features/kube_features.go

```diff
@@ -32,14 +32,23 @@ const (
 	// beta:  v1.19
 	//
 	// Enable legacy behavior to vary cluster functionality on the node-role.kubernetes.io labels. On by default (legacy), will be turned off in 1.18.
+	// Original copy from k8s.io/kubernetes/pkg/features/kube_features.go
 	LegacyNodeRoleBehavior featuregate.Feature = "LegacyNodeRoleBehavior"
 
 	// owner @brendandburns
 	// alpha: v1.9
 	// beta:  v1.19
 	//
 	// Enable nodes to exclude themselves from service load balancers
+	// Original copy from k8s.io/kubernetes/pkg/features/kube_features.go
 	ServiceNodeExclusion featuregate.Feature = "ServiceNodeExclusion"
+
+	// owner: @khenidak
```

Comment:
cc @khenidak we have to update this feature gate as well when we promote dual-stack to beta